CriticalMass/RequestManager.swift

```diff
@@ -27,6 +28,8 @@ public class RequestManager {
     private var networkLayer: NetworkLayer
     private var idProvider: IDProvider
 
+    private var log = OSLog(subsystem: Bundle.main.bundleIdentifier!, category: "RequestManager")
```

Comment:
Maybe the category can be more generic like „Networking“ and then outsource it in an extension of `OSLog` so it does not need to be an extra property of the RequestManager. But just a thought. Logger itself is nice. Just thought it is somehow strange to have the log in the class and then call have that for every class

Reply:
meant someting like:

```swift
extension OSLog {
    private static var subsystem = Bundle.main.bundleIdentifier!

    static let network = OSLog(subsystem: subsystem, category: "Network")
}
```

and then:

```swift
Logger.log(.info, log: .network, "Successfully finished API update")
```

But it is just an opinion.
Merge at will

Reply:
Good point, but let's do that once we need to add more logging in the future and adding the abstraction is needed